docs/supported-technologies.asciidoc

```diff
@@ -125,7 +125,7 @@ rendering
  This comes at the cost of increased startup times, however.
 
  Note: JAX-RS is only supported when running on a supported <<supported-app-servers, Application Server/Servlet Container>>.
-|1.0.0
+|1.0.0, jakarta.ws.rs since 1.28.0
```

Comment:
```suggestion
|1.0.0, `jakarta.ws.rs` since 1.28.0
```

Reply:
I believe we're supporting more recent versions than 2.x now. Please also check the other technologies if the supported versions are still up-to-date.